src/exponential_utils.jl

```diff
@@ -597,8 +597,8 @@ function phiv_timestep!(U::AbstractMatrix{T}, ts::Vector{tType}, A, B::AbstractM
   verbose && println("Absolute tolerance: $abstol")
   if iszero(tau)
     Anorm = opnorm(A, Inf)
-    b0norm = norm(@view(B[:, 1]), Inf)
-    tau = 10/Anorm * (abstol * ((m+1)/e)^(m+1) * sqrt(2*pi*(m+1)) /
+    b0norm = maximum(abs.(@view(B[:, 1])))
```

Comment:
This will allocate a new vector.

Reply:
Ah shoot, I was still under the impression that the `norm` here is the user-specified one, and going back to use `ODE_DEFAULT_NORM` doesn't work because of the `Inf`.